### ISSUE TITLE
Begin playing music after wake up

### ISSUE THREAD

**Issue by Yeti-or** [beardedspice/beardedspice#336]
I stop music
close mac
mac go to hibernate
after wake up music begin to play
osx 10.10.05
beardedspice/1.2.3

**Comment by Coder-256** [comment]
When your computer goes to sleep, it automatically pauses most processes, including BeardedSpice and your web browser. When it awakes, these processes are resumed. Your music should continue where you left off (unless it is live). What is the problem?

**Comment by Coder-256** [comment]
If you don't want this, you _would_ be able to change this behavior by stopping the music on wake using [SleepWatcher](http://www.bernhard-baehr.de/), [Scenario](http://www.lagentesoft.com/scenario/index.html), [Power Manager](https://www.dssw.co.uk/powermanager/) or similar. However, there must first be an API for BeardedSpice made available. See #332.

**Comment by Coder-256** [comment]
@Yeti-or Did that fix the problem?

**Comment by Stillness-2** [comment]
> I stop music

В чем музыка то играла? браузер/сайт? или там iTunes?

**Comment by chriswait** [comment]
@Coder-256: I think you may have misunderstood the original issue.

> I *stop music*
> close mac

If the music is paused before the mac is closed (and sleeping), we wouldn't expect it to start playing automatically upon waking (i.e this is unexpected behaviour). This seems unrelated to the resuming of processes, and might not be resolved using a B.S API, as you suggest.

**Comment by Coder-256** [comment]
@chriswait Yes, you might be right about the music being stopped before hibernating. However, this would still be fixable with a BS API, as I described. @Yeti-or, it would be helpful to know more information such as logs, browser version, and website.

**Comment by aray** [comment]
I have this problem also with BS 2.2.1 and Pandora Plus. Only happens when BS is running so definitely some sort of bug.

**Comment by 1ps0** [comment]
It's part of the automatic handling. I'll see if I can add a workaround.
